src/vs/workbench/contrib/tasks/browser/taskTerminalStatus.ts

```diff
@@ -59,7 +59,7 @@ export class TaskTerminalStatus extends Disposable {
 		});
 		problemMatcher.onDidFindErrors(() => {
 			if (this._marker) {
-				terminal.addGenericMark(this._marker, { hoverMessage: nls.localize('task.watchFirstError', "Beginning of detected errors for this run"), disableCommandStorage: true });
+				terminal.addBufferMark(Date.now().toString(), this._marker, false);
```

Comment:
Another case for undefined ids since we would never use this date id

Reply:
true, was using for testing 👍🏼